enterprise/web.core/src/org/netbeans/modules/web/jspcompiler/SmapResolver.java

```diff
@@ -318,10 +319,10 @@ public boolean isResolved() {
     /** 
      * get all the filenames in the SMAP
      */
-    public Map getFileNames() {
-        Hashtable h = new Hashtable(fsection.size());
-        Collection c = fsection.values();
-        Iterator i = c.iterator();
+    public Map<Integer, String> getFileNames() {
+        Map<Integer, String> h = new HashMap<>(fsection.size());
```

Comment:
This changes the semantic of the returned map. Is this a safe refactoring? Where is this map used?

Reply:
I tried to double check, but I could not find a caller of that method.

The only usage of `SmapResolver` I could find is in `JspJavacAntLogger`.

For testing purposes I removed the method `getFileNames` and did a `ant -Dcluster.config=enterprise build-nozip`.

Do you think this is enough evidence to remove the method entirely?

Reply:
Verified. The class is not exported API (and this modules declares one). So users of this class would need to have an implementation anyway. With that in mind, I see no problem keeping the methdo as you redesigned it.